src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -135,54 +128,45 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     });
   }, [selectedDayIndex, firstDayOfSchedule, isScheduleTimeshifted]);
 
-  const getEventLocation = useCallback(
-    (locString: string): VenueLocation => {
-      const [venueId, roomTitle] = extractLocation(locString);
-      const venueName = relatedVenues.find((venue) => venue.id === venueId)
-        ?.name;
-      return { venueId, venueName, roomTitle: roomTitle || undefined };
-    },
-    [relatedVenues]
-  );
-
-  const schedule: ScheduleDay = useMemo(() => {
+  const scheduleNG: ScheduleNGDay = useMemo(() => {
     const startOfSelectedDay = addDays(firstDayOfSchedule, selectedDayIndex);
+    const prioritiseEvents = (
+      a: WithVenueId<VenueEvent>,
+      b: WithVenueId<VenueEvent>
+    ) => {
+      if (a.start_utc_seconds !== b.start_utc_seconds) {
+        return a.start_utc_seconds - b.start_utc_seconds;
+      }
+
+      return a.duration_minutes - b.duration_minutes;
+    };
```

Comment:
i think it'd better to move the logic to the `utils/event` file and call it something like: `compareEventTime` ? `eventTimeComparator`? or something similar?

Reply:
set as `eventTimeConparator` in `utils/event` -> fixed in 88cf347e64a94383ef89b344faa4b13956ea1725